.github/workflows/build-test-deploy.yml

```diff
@@ -359,8 +359,11 @@ jobs:
           name: ${{  env.NIX_CACHIX }}
 
       - run: |
```

Comment:
```suggestion
      - timeout-minutes: 5
        run: |
```

Reply:
it will work only I split into 2 jobs as it was before. so her we have process, not docker now. will fix in next prs as soon as we decide about process manager https://composablefinance.slack.com/archives/C03MJD3E8KX/p1660655517102649